### ISSUE TITLE
[question] seed_input for CAM methods

### ISSUE THREAD

**Issue by ltetrel** [raghakot/keras-vis#168]
Hi everyone,

And thanks for this wonderfull tool. I am using it to better understand the different ways to interpret a CNN :)

So I was wondering, for Class Activation Maps, could we seed a random noise and let it converge until it maximizes the activation for one specific class ? Using an input image (like in `visualize_cam`) helps him to converge then ? 
I really see this problem as a generative model, so we could also use a regularizer on the loss (from a trained GAN for example, or other generative model).

Thanks,
